format/spec.md

```diff
@@ -458,11 +457,11 @@ The snapshot then populates the total number of `added-rows` based on the sum of
 When the new snapshot is committed, the table's `next-row-id` must also be updated (even if the new snapshot is not in the main branch). Because 225 rows were added (`added1`: 100 + `added2`: 0 + `added3`: 125), the new value is 1,000 + 225 = 1,225:
 
 
-##### Enabling Row Lineage for Non-empty Tables
+##### Row Lineage for Non-empty, Upgraded Tables 
 
 Any snapshot without the field `first-row-id` does not have any lineage information and values for `_row_id` and `_last_updated_sequence_number` cannot be assigned accurately.  
 
-All files that were added before `row-lineage` was enabled should propagate null for all of the `row-lineage` related
+All files that were added before upgrading to v3 should propagate null for all of the `row-lineage` related
```

Comment:
I think the `should` here in `should propogate null` has to be a `must` if I'm not mistaken? To not leave any room for implementations to propogate any non-null values for files that didn't have lineage.

Reply:
I've updated these references, but there may also be some follow up changes necessary for initial assignment.